### DISCUSSION TITLE
There is no docs about --extensions argument of @babel/cli

### DISCUSSION THREAD

**Discussion by krutoo** [babel/babel#13506] [Q&A] [answered]
Hi to all, i read docs in https://babeljs.io/docs/en/babel-cli

And There is no docs about --extensions argument, where can i read about all possible arguments (except help)?

**Comment by JLHwung** [comment, marked as answer]
The docs covers basic usage. Though we could add something about `--extensions`, at this time you can check out the cli usage using `babel -h`
```
  -x, --extensions [extensions]               List of extensions to compile when a directory has been the input.
                                              [.js,.jsx,.es6,.es,.mjs,.cjs]
```

You can also checkout https://babel.dev/docs/en/options for all available options. But at that point I would suggest using a dedicated `babel.config.json` for IDE auto-completions.
